### PR TITLE
Fix: Outdated image name in README

### PR DESCRIPTION
**Pull request by reitzig**
## What/Why/How?

Given command didn't work; using current image name now.

### CODE REVIEW

**Comment by adamaltman** [comment]
How about we fix the image name? @tatomyr ?

**Comment by RomanHotsiy** [comment]
@tatomyr I pushed new package: https://hub.docker.com/repository/docker/redocly/cli

I think it would be enough to just update the github action.

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ To build and run with a local image, run the following from the project root:
 
 ```
 docker build -t redocly-cli .
-docker run --rm -v $PWD:/spec redocly-cli lint path-to-root-file.yaml
+docker run --rm -v $PWD:/spec redocly/openapi-cli lint path-to-root-file.yaml
```
Comment (tatomyr) [inline README.md:67]:
```suggestion
docker run --rm -v $PWD:/spec redocly/cli lint path-to-root-file.yaml
```

**Comment by tatomyr** [comment]
> I think it would be enough to just update the github action.
I believe we have to adjust the other docs accordingly as well.

**Comment by AntonKozachuk** [comment]
Hi @reitzig,
Many thanks for your contribution to the project!
I created a new PR based on this one with a few small changes: 
https://github.com/Redocly/redocly-cli/pull/949
Should be published in the next release.

**Pull request closed without merging**